docs/gadget-devel/gadget-wasm-api-raw.md

```diff
@@ -567,3 +567,16 @@ Parameters:
 
 Return value:
 - (u32) 1 if the symbol exists, 0 otherwise.
+
+
+### Filtering
+
+#### `shouldDiscardMntnsID(mntnsID uint64) uint32`
+
+Check if a mount namespace ID should be discarded.
```

Comment:
Should it say that it can only be called at gadgetStart or after. If called at gadgetInit, the result might be undefined?